community-operators/microcks/microcks-operator.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: microcks
+channels:
+- name: alpha
+  currentCSV: microcks-operator.v0.1.0
```

Comment:
nit: Newline

Reply:
Do you mean EOF new line?

Reply:
Yes

Reply:
Ok. Added in commit caddfc74bd5023fc82628e70d9ca8bd08a643bc8